crates/telio-pmtu/src/lib.rs

```diff
@@ -0,0 +1,138 @@
+#[cfg(any(target_os = "linux", target_os = "android"))]
+#[path = "linux.rs"]
+mod platform;
+
+use std::{net::IpAddr, ops::Range, sync::Arc, time::Duration};
+
+use tokio::task::JoinHandle;
+
+#[cfg(any(target_os = "linux", target_os = "android"))]
+pub use platform::PMTUSocket;
+
+// We assume that 1000 is always supported.
+//1500 is the Ethernet MTU so we're not expecting PMTU to be higher than that.
+#[allow(unused)]
+const PMTU_RANGE: Range<u32> = 1000..1501;
+
+#[allow(unused)]
+const ICMP_ECHO_REQ_IDENT: u16 = 0x6C74; // "lt" (libtelio) in hex
+
+#[allow(unused)]
+const INTERVAL: Duration = Duration::from_secs(10 * 60); // 10 mins
```

Comment:
I am thinking, that this may be slightly too frequent for our needs, and can generate a bit too much quality metrics traffic when is really needed. Maybe lets make it 1h?

Reply:
Done